### DISCUSSION TITLE
how can i add multiple containers?

### DISCUSSION THREAD

**Discussion by byeongjokim** [milvus-io/milvus#27917] [Q&A and General discussion]
i want to attach custom container on pods(nodes and coords) for sending logging external.

in components global spec [crds](https://github.com/milvus-io/milvus-operator/blob/main/docs/CRD/milvus.md#components-global-spec), there is no options for attaching custom container(sidecar container).

are there other ways?

**Comment by yhmo** [comment]
Wait @LoveEachDay comment.
